scripts/pylib/twister/expr_parser.py

```diff
@@ -273,6 +273,14 @@ def ast_expr(ast, env, edt):
         if node and node.status == "okay":
             return True
         return False
+    elif ast[0] == "dt_node_prop_enabled":
```

Comment:
kind of odd to have `nodelabel` all one word and then `node_prop` with an underscore, but probably a nit :)

Reply:
@karben nodellabel is the label of the node, but node_prop is a property that belongs to the node.

Reply:
Yes, of course. I was commenting on the spelling as it's not clear why one would have an underscore and the other wouldn't.

Reply:
@hakehuang so what's your take then? It seems inconsistent to have `dt_nodelabel_enabled` (no underscore) and `dt_node_prop_enabled` (underscore)?
cc @decsny

Reply:
I think the underscore is fine. In zephyr DT vocabulary, "nodelabel" has become a valid single word over the years. Have never seen any occurence of people talking about a "nodeprop". So having the underscore is more natural feeling to me as a developer. Is there any logic to it, I don't know, but underscore feels fine to me, and not a big deal here.